src/workloads/query/multiplanner/NonBlockingVsBlocking.yml

```diff
@@ -1,13 +1,20 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/query"
 Description: |
-  The goal of this test is to exercise multiplanning. If the selectivity value is small enough (less
-  than 0.5), the optimal plan is to employ a blocking plan by scanning a segment of empty data and
-  conducting a blocking-sort operation, whereas the other plans' index provides the right sort
-  order, but requires a full scan, and every document is rejected after the FETCH stage. Because the
-  SBE multiplanner can't round-robin, it has a heuristic "try nonblocking plans first".  This
-  scenario is a worst case for that heuristic, because we'll try the best plan last. Otherwise, an
-  IXSCAN and FETCH non-blocking plan will be used.
+  The goal of this test is to exercise multiplanning when both blocking and non-blocking plans are
+  available.
+
+  If the selectivity value is small enough (less than 0.5), the optimal plan is to employ a
+  blocking plan by scanning a segment of empty data and conducting a blocking-sort operation,
+  whereas the other plans' index provides the right sort order, but requires a full scan, and
+  every document is rejected after the FETCH stage.
```

Comment:
Wait, every document is rejected? Unless I'm misreading the workload it looked like some of the data will match.

Reply:
Ping!

Reply:
I had put this on the wrong branch--should be fixed now.